internal/cmd/keys.go

```diff
@@ -208,8 +215,12 @@ func newKeysRemoveCmd(cli *CLI) *cobra.Command {
 			}
 
 			if len(keys) == 0 {
+				errMsg := fmt.Sprintf("No access key named '%s' for the current user, another user can be specified with the '--user' flag", keyName)
```

Comment:
```suggestion
				errMsg := fmt.Sprintf("No access key named %q for the current user. If trying to remove access key from another user, use the '--user' flag", keyName)
```

Reply:
I wonder if it would be worth it to look up the username and say:

> Access key {keyName} for user {user.Name} does not exist

(similar to the case below)

And maybe something like

> Use the '--user' flag to remove an access key for a different user.